provider/nativeDocs.go

```diff
@@ -1,6 +1,13 @@
 package provider
 
-import _ "embed" // nolint:golint
+import (
+	_ "embed" // nolint:golint
+)
+
+//go:embed pkg/docs-gen/image-description.md
+var docImageDescription string
 
 //go:embed pkg/docs-gen/examples/image.md
-var docImage string
+var docImageExamples string
+
+var docImage string = docImageDescription + "\n\n{{% examples %}}\n" + docImageExamples + "{{% /examples %}}"
```

Comment:
Rather than setting the shortcode manually here should we set it in [generate.go?](https://github.com/pulumi/pulumi-docker/blob/master/provider/pkg/docs-gen/examples/generate.go#L55) - that would then get used in all other native resources going forward, and is safe to do now that we removed the extraneous shortcode gen from the bridge.